gcp/rakefiles/utils.rb

```diff
@@ -0,0 +1,13 @@
+# Obfuscate sensitive data in the output of a command, relying on @secrets
+def sh_filter(*cmd)
+  if @secrets
+    IO.popen(*cmd).each do |out|
+      @secrets.each do |key, val|
+        out = out.gsub("#{val}", "<SENSITIVE>")
```

Comment:
I believe we should use [Regexp.escape](https://ruby-doc.org/core-2.4.3/Regexp.html#method-c-escape) on `val` here.

(I was going to point this out for the `sed`-based strategy too. I think the ruby solution is cleaner :).)